### PR TITLE
Throw error for Netty when Jetty ALPN not configured.

### PR DESCRIPTION
**Pull request by nmittler**

### CODE REVIEW

**Comment by nmittler** [comment]
@ejona86 PTAL

File: netty/src/main/java/io/grpc/transport/netty/JettyAlpnVerifier.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package io.grpc.transport.netty;
+
+/**
+ * Utility class that verifies that Jetty ALPN is properly configured for the system.
+ */
+public final class JettyAlpnVerifier {
```
Comment (ejona86) [inline netty/src/main/java/io/grpc/transport/netty/JettyAlpnVerifier.java]: Is there a reason this is public?

Comment (nmittler) [inline netty/src/main/java/io/grpc/transport/netty/JettyAlpnVerifier.java]: Hmm ... I'm throwing an exception that is an inner class.  I suspect odd things may occur if the exception is not public.

**Comment by nmittler** [comment]
@ejona86 PTAL

**Comment by ejona86** [comment]
@nmittler LGTM

**Comment by nmittler** [comment]
Cherry-picked as 4322a43

**Pull request closed without merging**